### PR TITLE
Update students.yml

### PR DESCRIPTION
**Pull request by lavavi**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

**Comment by lavavi** [comment]
lavavi

**Comment by abishekvashok** [comment]
Please add your image in 240x240px to /img/students folder and commit. If possible do a got amend commit so that you will have only one commit in the pr. In addition to that update the PR body as well :)

Cheers.

**Comment by anshumanv** [comment]
@abishekvashok the image seems to be 240x240, can you please investigate why travis is failing?

**Comment by the-codingninja** [comment]
You should check this log
https://travis-ci.org/fossasia/gci17.fossasia.org/jobs/315946160

File: _data/students.yml [hunk 1]
```diff
@@ -156,3 +156,9 @@
   country: Poland
   github: Shizerq
   image: MSShizerq.jpg
+  
+- name: Ashish Kumar Singh
+  country: India
+  github: lavavi
+  image: lavavi.jpg
+  twitter: @AshishK51681851
```
Comment (2coolife) [inline _data/students.yml]: The '@' character is not needed here :)

Comment (ryushar) [inline _data/students.yml]: The '@' character is not needed here :)

**Review by ryushar — CHANGES_REQUESTED** [review]

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Make the suggested changes and try to amend the commit and force push it. Refer this for help, https://help.github.com/articles/changing-a-commit-message/ :smile:

**Review by akshatnitd — CHANGES_REQUESTED** [review]
As you have already made some contributions, I would request you to squash the commits. Please refer this for help, https://gist.github.com/patik/b8a9dc5cd356f9f6f980

**Review by ParthS007 — COMMENTED** [review]
@lavavi Your PR is ready to merge but please try to squash commits and let us know If you are facing difficulty. 👍

**Review by anshumanv — APPROVED** [review]

**Comment by anshumanv** [comment]
Try to squash commits from next time.

**Comment by lavavi** [comment]
ok

**Pull request merged**